datacite-update-doi-handler/src/main/java/no/unit/nva/datacite/handlers/UpdateDoiEventHandler.java

```diff
@@ -125,6 +126,19 @@ private void handleDoi(DoiStateDto doiStateDto, DoiUpdateRequestEvent input, Doi
 
     }
 
+    private void handleRegisteredDoi(DoiUpdateRequestEvent requestEvent, Doi doi) {
+        logger.info(DOI_ALREADY_REGISTERED_MESSAGE, doi, requestEvent.getCustomerId(), requestEvent.getPublicationId());
+    }
+
+    private void handleDraftDoi(DoiUpdateRequestEvent updateRequestEvent, Doi doi) {
+        try {
+            logger.info(DELETING_DRAFT_DOI_MESSAGE, doi.getUri(), updateRequestEvent.getCustomerId(), updateRequestEvent.getPublicationId());
+            doiClient.deleteDraftDoi(updateRequestEvent.getCustomerId(), doi);
+        } catch (ClientException ex) {
+            throw new RuntimeException(ex);
+        }
+    }
```

Comment:
Should we remove DOI from publication? Emit event to publication-api?

Reply:
not fun but probably something we should do yes, or ask JE again if hes absoooolutely sure we need to delete doi :-P